index.js

```diff
@@ -5,6 +5,8 @@ const got = require("got");
 const assert = require("assert");
 const { CookieJar } = require("cookiejar");
 
+const isJson = /application\/json/i;
```

Comment:
prefix `is` makes it sound like the result of a type evaluation.
maybe `jsonContentTypePattern`. 
🤷